### PR TITLE
Index List

### PR DESCRIPTION
**Pull request by cutandpastey**
screen shot 2015-04-07 at 15 39 03

### CODE REVIEW

File: sass/components/index-list.scss [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+.index-list {
+  @include index-list;
+}
+
+.index-list__item {
+  @include index-list__item;
+}
+
+.index-list__item__index {
```
Comment (jamespamplin) [inline sass/components/index-list.scss:13]: Is this BEM?

Perhaps `.index-list-item__index` (list item as the block) or `.index-list__item-index` (index-list is still the block, with element item-index).

Mixin files should be renamed to match also.

File: app/templates/index-list.html [hunk 2]
```diff
@@ -0,0 +1,15 @@
+<gu-row>
+  <gu-column span="4">
+    <app-demo>
+      <h2>Index List</h2>
+      <app-demo-code>
+        <gu-index-list>
+          <gu-index-list-item index="1" variant="primary">Content</gu-index-list-item>
+          <gu-index-list-item index="2" variant="secondary">Content</gu-index-list-item>
+          <gu-index-list-item index="3" variant="tertiary">Content</gu-index-list-item>
+          <gu-index-list-item index="3" variant="quaternary">Content</gu-index-list-item>
```
Comment (jamespamplin) [inline app/templates/index-list.html]: Index=4 for "quaternary"

**Comment by jamespamplin** [comment]
A couple of minor things, but cool stuff :+1:

File: sass/components/index-list.scss (hunk 1 above)
Comment (cutandpastey) [inline sass/components/index-list.scss:13]: I think what you are suggesting would only work if `index-list__item` was not a block, which sadly it is, therefore you get `index-list__item__index`

**Pull request merged**